client/ReviewListItem.jsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+const moment = require('moment');
 import styles from './styles/reviewListItem.css';
 
 class ReviewListItem extends React.Component {
```

Comment:
consider making it a stateless component instead of a stateful component if state isn't being utilized

Reply:
Refactor goal #3!